candle-nn/tests/varlen_attention.rs

```diff
@@ -0,0 +1,2235 @@
+use candle::Result;
+use candle_nn::varlen_attention::flash_attn_varlen_unfused;
+
+const FA_FEATURE_ENABLED: bool = false; // flash-attn features not available in this workspace
```

Comment:
can't have FA2 kernel in this repo, if i were to have them, I could run the below tests.